src/pages/resources/mainnet/upgrades/v1.0.mdx

```diff
@@ -0,0 +1,74 @@
+# Mainnet upgrade: v1.0
+
+import { Callout } from '/src/components/callout'
+
+## About the v1.0 upgrade
+
+This release finalizes initial support for [Amplifier](/dev/amplifier/introduction).
+
+Changes since 0.35.x:
+- Various fixes that improve stability
+- New queries in the axelarnet module to query the IBC path for a chain and the chain by IBC path
+- Added a new message type “BatchRequest”. In contrast to cosmos default behaviour, where the entire tx gets rolled back if there is an error in any of the contained message executions, this batch only rolls back the state for the specific messages that fail and keeps the rest
+- Allow refundable messages to be batched
+- Correctly pass message IDs between core and the new amplifier
+- Add functionality to quickly (de)activate the connection between core and amplifier in case of emergencies
+
+See all the changes in the [`axelar-core` release notes](https://github.com/axelarnetwork/axelar-core/releases).
+
+Upgrade height `14216700` at approximately **3:00PM UTC on August 27th, 2024** ([countdown](https://www.mintscan.io/axelar-testnet/block/14625900)) 
+
+<Callout type="warning" emoji="⚠️">
+As of v1.0.0, the `axelard` [binary releases](https://github.com/axelarnetwork/axelar-core/releases) require `GLIBC_2.34`, which is not present on Ubuntu 20.04.
```

Comment:
```suggestion
As of v1.0.1, the `axelard` [binary releases](https://github.com/axelarnetwork/axelar-core/releases) require `GLIBC_2.34`, which is not present on Ubuntu 20.04.
```
just in case people misunderstand this as upgrading to v1.0.0